reversed_func.py

```diff
@@ -0,0 +1,21 @@
+def reversed_func(text: str) -> str:
+
```

Comment:
Where did docstring go? I loved it

Reply:
Sorry, I've lost it accidentally. I fixed and commited it.